File: EvChargeHelper/app/src/main/java/com/chebuso/chargetimer/settings/ISettingsReader.java
package com.chebuso.chargetimer.settings;

public interface ISettingsReader {
    boolean firstApplicationRun();

    boolean applicationNotificationsAllowed();

    boolean calendarBasicNotificationsAllowed();

    boolean calendarAdvancedNotificationsAllowed();

    double getBatteryCapacity();

    int getChargingLossPct();

    int getDefaultAmperage();

    int getDefaultVoltage();

    int getApplicationReminderMinutes();

    int getCalendarReminderMinutes();
}

File: EvChargeTimer/app/src/main/java/com/chebuso/chargetimer/settings/ISettingsReader.kt
package com.chebuso.chargetimer.settings


interface ISettingsReader {
    fun calendarBasicNotificationsAllowed(): Boolean
    fun calendarAdvancedNotificationsAllowed(): Boolean
    fun getCalendarReminderMinutes(): Int
    fun getDefaultAmperage(): Int
    fun getDefaultVoltage(): Int
    fun applicationNotificationsAllowed(): Boolean
    fun getApplicationReminderMinutes(): Int
    fun getBatteryCapacity(): Double
    fun getChargingLossPct(): Double
    fun firstApplicationRun(): Boolean
}

